docs/whatisk8s.md

```diff
@@ -0,0 +1,48 @@
+# What is Kubernetes? 
+
+Kubernetes is an open-source platform for automating deployment, scaling, and operations of application containers across clusters of hosts. 
+
+Kubernetes builds upon a [decade and a half of experience that Google has with running production workloads at scale](https://research.google.com/pubs/pub43438.html), combined with best-of-breed ideas and practices from the community.
+
+With Kubernetes, you are able to quickly and efficiently respond to customer demand: 
+
+ - Scale your applications on the fly.
+ - Seamlessly roll out new features.
+ - Optimize use of your hardware by using only the resources you need.
+
+#### Kubernetes is:
+
+* **lean**: lightweight, simple, accessible
+* **portable**: public, private, hybrid, multi-cloud
+* **extensible**: modular, pluggable, hookable, composable
+* **self-healing**: auto-placement, auto-restart, auto-replication
+
+The Kubernetes project was started by Google in 2014.
```

Comment:
Doesn't make sense here. Why not just copy my changes from gh-pages. I can do it if it would be easier